src/annotator/test/guest-test.js

```diff
@@ -262,55 +279,82 @@ describe('Guest', () => {
       it('scrolls to the anchor with the matching tag', () => {
         const highlight = document.createElement('span');
         const guest = createGuest();
+        const fakeRange = sinon.stub();
         guest.anchors = [
-          { annotation: { $tag: 'tag1' }, highlights: [highlight] },
+          {
+            annotation: { $tag: 'tag1' },
+            highlights: [highlight],
+            range: new FakeTextRange(fakeRange),
+          },
         ];
+
         emitGuestEvent('scrollToAnnotation', 'tag1');
+
         assert.called(scrollIntoView);
         assert.calledWith(scrollIntoView, highlight);
       });
 
-      context('when dispatching the "scrolltorange" event', () => {
-        it('emits with the range', () => {
-          const highlight = document.createElement('span');
-          const guest = createGuest();
-          const fakeRange = sinon.stub();
-          guest.anchors = [
-            {
-              annotation: { $tag: 'tag1' },
-              highlights: [highlight],
-              range: fakeRange,
-            },
-          ];
-
-          return new Promise(resolve => {
-            guest.element.addEventListener('scrolltorange', event => {
-              assert.equal(event.detail, fakeRange);
-              resolve();
-            });
+      it('emits a "scrolltorange" DOM event', () => {
```

Comment:
This is just a tad confusing as the test _listens_ for `scrolltorange`.

"scrolltorange" is a funny event name.

📜 T 🍊